listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -40,3 +40,12 @@ def check_connection(self):
         except redis.exceptions.ConnectionError as e:
             self.log.error("Redis ping didn't work: {}".format(str(e)))
             raise
+
+    def store_latest_listens(self):
```

Comment:
This function should take a list of listens and put them inside redis.


```suggestion
    def store_latest_listens(self, listens):
        self.redis.set('latest_listens', listens)
```